driver.go

```diff
@@ -98,6 +98,27 @@ func (hd *hetznerDriver) Create(req *volume.CreateRequest) error {
 		}
 	}
 
+	uid := getOption("uid", req.Options)
+	gid := getOption("gid", req.Options)
+	if uid != "0" || gid != "" {
+		// string to int
+		uintParsed, err := strconv.Atoi(uid)
+		if err != nil {
+			logrus.Warnf("failed to parse uid option value as uint32: %s", uid)
+			uintParsed = 0
+		}
+		gidParsed, err := strconv.Atoi(gid)
+		if err != nil {
+			logrus.Warnf("failed to parse gid option value as uint32: %s", gid)
+			gidParsed = 0
+		}
```

Comment:
I think we have an interesting error scenario here: if the user sets uid/gid to "foo", they effectively cause a `chown root:root`. This may not be the best option. Shouldn't we just skip chowning completely on parse errors instead?

Reply:
Ok, this is a good point. I will adapt the PR accordingly